include/pmacc/memory/buffers/ExchangeIntern.hpp

```diff
@@ -222,24 +222,24 @@ namespace pmacc
 
         }
 
-        virtual HostBuffer<TYPE, DIM>& getHostBuffer()
+        virtual HostBuffer<TYPE, DIM>& getHostBuffer() override
```

Comment:
I think now with `override` there is no need in `virtual`. Technically it was not needed even before, but then it shows the function was virtual, however now override already guarantees it.

Reply:
I will remove it